mkdocs.yml

```diff
@@ -8,19 +8,33 @@ nav:
     - overview/policy_issues.md
     - overview/outreach.md
   - Focus Countries:
+    - World Map: countries/worldmap.md
+    - Burkina Faso:
+      - countries/burkina_faso/background.md
```

Comment:
get rid of the dash in `countries/burkina_faso/background.md` and put on same line as "Burkina Faso"